src/components/Auth/Login/Login.react.js

```diff
@@ -311,6 +313,18 @@ class Login extends Component {
 									labelColor="#fff" />
 							</Link>
 						</div>
+						<h4 style={{
+							margin: '8px 0'
+						}}><Translate text="OR"/></h4>
+                    <FacebookLogin socialId="392360367885660"
+                       language="en_US"
+                       scope="public_profile,email"
+                       responseHandler={this.responseFacebook}
+                       xfbml={true}
+                       fields="id,email,name"
+                       version="v2.5"
+                       className="facebook-login btn btn-primary"
```

Comment:
Please remove bootstrap classes and import FB icon from material UI library. See how other icons are being imported for reference. The icon can be found here: https://www.iconfinder.com/
Make sure you are using free icons.